atlasdb-config/src/main/java/com/palantir/atlasdb/factory/TransactionManagers.java

```diff
@@ -792,8 +794,13 @@ private static LockAndTimestampServices getLockAndTimestampServices(
             String userAgent) {
         LockService lockService = new ServiceCreator<>(metricsManager, LockService.class, userAgent)
                 .applyDynamic(timelockServerListConfig);
-        TimelockService timelockService = new ServiceCreator<>(metricsManager, TimelockService.class, userAgent)
-                .applyDynamic(timelockServerListConfig);
+
+        TimelockServerInterface rawTimelockService = new ServiceCreator<>(metricsManager,
```

Comment:
maybe worth calling this TimelockRpcs or something like that? TimelockServerInterface seems a bit fixable...